### PR TITLE
Feat: Sanitise query

### PR DESCRIPTION
**Pull request by techsavvyash**
- [x] Table hash resolution
- [x] SQL Query Injection Fixes

### CODE REVIEW

File: src/maps/whitelist.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+  "queries": [
+    "select grade from dimensions.grade",
+    "select grade from dimensions.grade",
```
Comment (ChakshuGautam) [inline src/maps/whitelist.json]: duplicate

**Review by ChakshuGautam — COMMENTED** [review]
Fix duplicates in this.

File: src/maps/whitelist.json (hunk 1 above)
Comment (techsavvyash) [reply src/maps/whitelist.json]: fixed

File: src/database/database.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }).compile();
+
+    service = module.get<DatabaseService>(DatabaseService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+
+  it('should change he query text', async () => {
```
Comment (ChakshuGautam) [inline src/database/database.service.spec.ts]: Can you add a test for whitelisted queries as well?

Comment (techsavvyash) [reply src/database/database.service.spec.ts]: done

**Pull request merged**